### PR TITLE
DNN-8639

### PR DESCRIPTION
**Pull request by RichardMartens**
Hello.

I deleted the references to missed files in Website project.

I didn't sign Contributor License Agreement, because I didn't find where i can do it.

Regards,
Richard

### CODE REVIEW

File: DNN Platform/Tests/DotNetNuke.Tests.Web.Mvc/DotNetNuke.Tests.Web.Mvc.csproj [hunk 1]
```diff
@@ -117,6 +117,9 @@
       <Name>DotNetNuke.Tests.Utilities</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```
Comment (zyhfish) [inline DNN Platform/Tests/DotNetNuke.Tests.Web.Mvc/DotNetNuke.Tests.Web.Mvc.csproj]: this line looks not needed.

File: Website/Properties/PublishProfiles/local.pubxml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (zyhfish) [inline Website/Properties/PublishProfiles/local.pubxml]: this files looks not needed for this issue.

**Comment by ashishpd** [comment]
@zyhfish can you please clean this up and submit a new PR. We will close this one.

**Comment by zyhfish** [comment]
merged, and remove un-need changes.

**Pull request merged**